tests/components/conditional_formatting.test.ts

```diff
@@ -611,7 +611,7 @@ describe("UI of conditional formats", () => {
     await click(fixture, selectors.buttonSave);
 
     const sheetId = model.getters.getActiveSheetId();
-    expect(dispatch).toHaveBeenCalledWith("ADD_CONDITIONAL_FORMAT", {
+    expect(dispatch).toHaveBeenNthCalledWith(1, "ADD_CONDITIONAL_FORMAT", {
```

Comment:
Why this change ? The test seems to work with just `toHaveBeenCalledWith ` , and we don't really care about the exact sequence order of dispatch